docs/index.md

```diff
@@ -2,32 +2,43 @@
 
 <p style="font-size: larger; font-weight: bold;">Transform your research with vast amounts of computing!</p>
 
-The OSG Virtual School 2021 will prepare you to use high throughput computing (HTC) systems&nbsp;&mdash;
-locally or using the OSG&nbsp;&mdash;
+The OSG Virtual School 2021 will prepare you to use high throughput computing (HTC)&nbsp;&mdash;
+whether on a 'local' system or using the 
+[Open Science Grid (OSG)](https://www.youtube.com/watch?v=t2PzIy-vvaE&list=PLBWb4iScSWcPy7LQ4BuXmm8Z2xO4ZW1J_)&nbsp;&mdash;
 to run large-scale research simulation, analysis, and other applications for cutting-edge science.
-Through lecture, demonstration, hands-on exercises, and one-on-one consulting with experienced OSG staff,
+Through lecture, demonstration, hands-on exercises, and peronalized consulting with experienced OSG staff,
 you will learn to harness HTC systems and apply them to your own research computing work.
-The event has been tuned for remote participation and is not merely a translation of the in-person program!
-
-The Virtual School will be held **the weeks of August 2&ndash;6 and 9&ndash;13**.
-You can view [the high-level schedule](schedule.md) now.
-
-Participation in the event is free, but a selective application process is required.
-The School is ideal for graduate students in any research domain
-where large-scale computing is a vital part of the research process,
-plus we will consider applications from advanced undergraduates, post-doctoral students, faculty, and staff.
-While no travel is involved and the schedule allows for some flexibility,
-the expectation is that participants commit to about 20 hours each week.
-
-**Applications will open in late April or early May;** check back here then.
-
-Note: We hope to return to an in-person OSG User School in 2022!
+The School is ideal for graduate students in any research area for whom beyond-the-desktop computing 
```

Comment:
Is this meant to start a new paragraph? Also, I think the term “beyond-the-desktop computing” has meaning to you, but not the world in general; and it is likely to be a more challenging neologism for non-native speakers. I aim to keep things simple…